tree/tree/src/TBasket.cxx

```diff
@@ -1096,7 +1096,7 @@ Int_t TBasket::WriteBuffer()
          // NOTE this is declared with C linkage, so it shouldn't except.  Also, when
          // USE_IMT is defined, we are guaranteed that the compression buffer is unique per-branch.
          // (see fCompressedBufferRef in constructor).
-         R__zipMultipleAlgorithm(cxlevel, &bufmax, objbuf, &bufmax, bufcur, &nout, cxAlgorithm);
+         R__zipMultipleAlgorithm(cxlevel, &bufmax, objbuf, &bufmax, bufcur, &nout, static_cast<ROOT::ECompressionAlgorithm>(cxAlgorithm));
```

Comment:
Would it make more sense to change the type of cxAlgorithm?

Reply:
I thought about both and either way involves a single `static_cast` ... is there a reason why you prefer it in `cxAlgorithm` over here?  Or am I missing something?

Reply:
It shortens this line a bit.

In another function this also reduce the number of time static_cast is type by one or two.

Mostly it makes more semantic sense (is cxAlgorithm is a ECompressionAlgorithm rather than a random int).